src/main/app.ts

```diff
@@ -47,8 +46,6 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended: false}));
 app.use(cookieParser());
 
-expressNunjucks(app);
```

Comment:
Potentially a breaking change by removing this line because Nunjucks won't be initialised for `case-data.ts` in `routes` - it doesn't have its own instance of a Nunjucks environment. @vlaurin - your take on this?

Reply:
Darn it! I thought I had added that so I removed it by mistake

Reply:
I've corrected the above